basemodels/__init__.py

```diff
@@ -33,6 +33,15 @@ class Manifest(Model):
     job_total_tasks = IntType()
 
     requester_restricted_answer_set = DictType(DictType(StringType))
+
+    def validate_requester_restricted_answer_set(self, data, value):
```

Comment:
I don't see where this function is called; automatically on manifest.validate()?

Reply:
Yep! That occurs in the tests and in the job creation in exchange